ipn/ipnlocal/profiles_test.go

```diff
@@ -681,8 +681,18 @@ func TestProfileBackfillStatefulFiltering(t *testing.T) {
 			// StatefulFiltering boolean.
 			pf := pm.CurrentPrefs()
 			if !pf.NoStatefulFiltering().EqualBool(tt.want) {
-				t.Fatalf("got NoStatefulFiltering=%v, want %v", pf.NoStatefulFiltering(), tt.want)
+				t.Fatalf("got NoStatefulFiltering=%q, want %v", pf.NoStatefulFiltering(), tt.want)
 			}
 		})
 	}
 }
+
+func TestDefaultPrefs(t *testing.T) {
```

Comment:
Thanks! Can you add a short description that this tests that the only difference is running and logged out to allow tailscaled to startup without calling out to control?